draft-ietf-webpush-protocol.xml

```diff
@@ -530,10 +558,11 @@ TTL = 1*DIGIT
       </t>
       <t>
         Each push message is pushed in response to a synthesized GET request.
-        The GET request is made to the push message resource that was created by
+        This GET request is made to the push message resource that was created by
         the push service when the application server requested message delivery.
-        The response body is the entity body from the most recent request sent
-        to the push resource.
+        The push service MUST include the link references to the push and receipt
+        subscribe resources in the request. The response body is the entity body

```

Comment:
I note also that the receipt resource isn't needed in the request, just the response and only optionally.  I think that we need to specially justify the inclusion of the link relation in the push promise as well.